src/components/BaseCommentForm.js

```diff
@@ -69,11 +75,22 @@ class BaseCommentForm extends React.Component {
   }
 
   render() {
+    const canSetNickname = this.props.canSetNickname;
     if (!this.state.collapsed) {
       return (<div className="comment-form">
         <form>
           <h3><FormattedMessage id="writeComment"/></h3>
           <Input type="textarea" value={this.state.commentText} onChange={this.handleTextChange.bind(this)}/>
+          {canSetNickname ? <h3><FormattedMessage id="nickname"/></h3> : null}
+          {canSetNickname ? (
+            <Input
+              type="text"
+              placeholder={this.props.intl.formatMessage({id: "anonymous"})}
```

Comment:
https://github.com/City-of-Helsinki/kerrokantasi-ui/blob/master/src/i18n/fi.json#L41